addon/templates/components/bs3/bs-form/element/layout/horizontal/checkbox.hbs

```diff
@@ -1,8 +1,7 @@
 <div class="{{horizontalInputGridClass}} {{horizontalInputOffsetGridClass}}">
   <div class="checkbox">
-    {{#component labelComponent}}
-      {{yield}}
-    {{/component}}
+    {{yield}}
+    {{component labelComponent}}
```

Comment:
Hm, I am not sure we should do this change also for BS3!? 
The old docs suggest to wrap the checkbox in a label there: https://getbootstrap.com/docs/3.3/css/#checkboxes-and-radios

As the suggested markup is different for BS4 (unwrapped), we could see a BS4-only change as a bug fix, thus (at least technically) not having to declare this a breaking change! 😉